sites/default/statement.inc.php

```diff
@@ -300,15 +325,53 @@ function create_statement($stmt) {
  }
  $out .= "\n";
  $out .= sprintf("%-s: %-25s %-s: %-14s %-s: %8s\n",$label_ptname,$stmt['patient'],
-                 $label_today,$stmt['today'],$label_due,$stmt['amount']);
+                 $label_today,oeFormatShortDate($stmt['today']),$label_due,$stmt['amount']);
  $out .= sprintf("__________________________________________________________________\n");
  $out .= "\n";
  $out .= sprintf("%-s\n",$label_call);
  $out .= sprintf("%-s\n",$label_prompt);
  $out .= "\n";
  $out .= sprintf("%-s\n",$billing_contact);
- $out .= sprintf("  %-s\n",$label_dept);
- $out .= sprintf("  %-s\n",$billing_phone);
+ $out .= sprintf("  %-s %-25s\n",$label_dept,$billing_phone);
+if($GLOBALS['statement_message_to_patient']) {
+ $out .= "\n"; 
+ $statement_message = $GLOBALS['statement_msg_text'];
+ $out .= sprintf("%-40s\n",$statement_message);
+}
+if($GLOBALS['show_aging_on_custom_statement']) {
+# code for ageing
+ $ageline .= ' / Over-' . ($age_index * 30) .
```

Comment:
Does the Over need to be translated?(I'll leave this decision to you, since I am not sure what this aging stuff will look like)